### PR TITLE
Update buy back and burn strategy

### PR DESCRIPTION
**Pull request by vovac12**

### CODE REVIEW

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: pallets/xor-fee/src/lib.rs [hunk 1]
```diff
@@ -239,16 +241,27 @@ where
                     referrer,
                     referrer_portion.into(),
                 ));
+            } else {
+                total_xor_to_vxor += referrer_xor.peek();
```
Comment (wer1st) [inline pallets/xor-fee/src/lib.rs:245]: checked or saturating add

Comment (wer1st) [reply pallets/xor-fee/src/lib.rs:245]: The same for all code below.
And above

File: runtime/src/tests/remote.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@ use std::env::var;
 async fn run_migrations() {
     sp_tracing::try_init_simple();
     let transport: Transport = var("WS")
-        .unwrap_or("wss://ws.framenode-0.a1.sora2.soramitsu.co.jp:443".to_string())
+        .unwrap_or("wss://ws.framenode-2.s1.stg1.sora2.soramitsu.co.jp:443".to_string())
```
Comment (wer1st) [inline runtime/src/tests/remote.rs:12]: Why?

Comment (wer1st) [reply runtime/src/tests/remote.rs:12]: Btw, it is better to introduce special consts for such cases.

**Review by wer1st — APPROVED** [review]

**Pull request merged**